google/cloud/bigtable/internal/data_connection_impl_test.cc

```diff
@@ -1474,6 +1471,42 @@ TEST_F(DataConnectionTest, SampleRowsPermanentError) {
   EXPECT_THAT(samples, StatusIs(StatusCode::kPermissionDenied));
 }
 
+TEST_F(DataConnectionTest, SampleRowsBigtableCookie) {
+  auto mock = std::make_shared<MockBigtableStub>();
+  EXPECT_CALL(*mock, SampleRowKeys)
+      .WillOnce(
```

Comment:
Why does this need to return twice?

Reply:
In the first request, the server gives us a cookie.

In the second request, we give that cookie back to the server.

Reply:
Added comments for this, and all other `*BigtableCookie` tests.